security/org-account-context/dependencies.tf

```diff
@@ -35,3 +35,14 @@ data "aws_iam_policy_document" "shared_role_cap_acc" {
     }
   }
 }
+
+data "aws_iam_policy_document" "assume_role_policy_master_account" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::${var.master_account_id}:root"]
```

Comment:
We should try to limit these trust relationships.

Reply:
Well, it is a requirement that the vendor Certero has from connecting from our master account to each organization account.

Reply:
Yeah, but we can limit it to the user/role they will utilize instead of `root`.